salt/metalk8s/orchestrate/deploy_salt_minion_on_new_node.sls

```diff
@@ -0,0 +1,23 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if 'bootstrap' in control_plane_ips.keys() and control_plane_ips['bootstrap'] %}
+{%- set control_plane_ip = control_plane_ips['bootstrap'][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+Test:
+  salt.state:
+    - ssh: true
+    - roster: kubernetes
+    - tgt: node1
```

Comment:
Should be passed in as an argument to `salt-run` (through pillar values).

Reply:
Done.